packages/shared/src/components/profile/ProfileButton.tsx

```diff
@@ -14,40 +14,38 @@ const ProfileMenu = dynamic(
 
 interface ProfileButtonProps {
   className?: string;
-  atMobileSidebar?: boolean;
+  settingsIconOnly?: boolean;
 }
 
 export default function ProfileButton({
   className,
-  atMobileSidebar,
+  settingsIconOnly,
 }: ProfileButtonProps): ReactElement {
   const { isOpen, onUpdate, wrapHandler } = useInteractivePopup();
   const { user } = useContext(AuthContext);
 
   return (
     <>
-      {atMobileSidebar ? (
+      {settingsIconOnly ? (
         <Button
-          iconOnly
-          className="btn btn-tertiary"
+          variant={ButtonVariant.Tertiary}
           onClick={wrapHandler(() => onUpdate(!isOpen))}
           icon={<SettingsIcon />}
         />
       ) : (
         <SimpleTooltip placement="left" content="Profile settings">
-          <button
-            type="button"
+          <Button
```

Comment:
After thinking, maybe this shouldn't even be a button tbh 😅
It's like a custom element with a onClick really.
But open to keep it for simplicity